### PR TITLE
Cloudformation templates for Kinesis connector , Kafka connector and windowing examples

### PR DESCRIPTION
**Pull request by akshayar**
*Issue #, if available:*

*Description of changes:*

1. Cloudformation templates and instructions
2. Updated instructions for execution from IntelliJ
3. Added instructions for execution from Maven
4. Separated Windowing into two samples WindowingSliding and WindowingTumbling. Both read from Kinesis. Removed Kafka windowing example. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by akshayar** [comment]
@jeremyber-aws , @FranMorilloAWS and @nicusX , request you for review.

File: java/KafkaConnectors/README.md [hunk 1]
```diff
@@ -32,7 +34,85 @@ Configuration parameters:
...
+
 
-### Running locally in IntelliJ
+## Running locally in IntelliJ
 
 To start the Flink job in IntelliJ edit the Run/Debug configuration enabling *'Add dependencies with "provided" scope to the classpath'*.
+
+Provide arguments like following -
+```
+--source.bootstrap.servers localhost:9092 --source.topic source --sink.bootstrap.servers localhost:9092 --sink.topic sink --sink.transaction.timeout.ms 1000
```
Comment (jeremyber-aws) [inline java/KafkaConnectors/README.md:63]: can you show how to add arguments in Intellij Environment properties? This is how most customers will do it and not run CLI with IDE

Comment (akshayar) [reply java/KafkaConnectors/README.md:63]: These are just argument. 
There are two options to make it better - 
1. I will show the screenshot of run configurations. 
2. I can give the xml run configuration like this - 
<component name="ProjectRunConfigurationManager">
  <configuration default="false" name="SlidingWindowStreamingJobWithParallelism" type="Application" factoryName="Application" nameIsGenerated="true">
    <option name="INCLUDE_PROVIDED_SCOPE" value="true" />
    <option name="MAIN_CLASS_NAME" value="com.amazonaws.services.msf.windowing.kinesis.SlidingWindowStreamingJobWithParallelism" />
    <option name="PROGRAM_PARAMETERS" value="--InputStreamRegion ap-south-1 --InputStreamName stream-input --OutputStreamRegion ap-south-1 --OutputStreamName stream-output" />
    <extension name="coverage">
      <pattern>
        <option name="PATTERN" value="com.amazonaws.services.msf.windowing.kinesis.*" />
        <option name="ENABLED" value="true" />
      </pattern>
    </extension>
    <extension name="software.aws.toolkits.jetbrains.core.execution.JavaAwsConnectionExtension">
      <option name="credential" />
      <option name="region" />
      <option name="useCurrentConnection" value="false" />
    </extension>
    <method v="2">
      <option name="Make" enabled="true" />
    </method>
  </configuration>
</component>%

Comment (akshayar) [reply java/KafkaConnectors/README.md:63]: I have added run configuration images. Does that work ?

File: java/KafkaConnectors/README.md [hunk 2]
```diff
@@ -32,7 +34,90 @@ Configuration parameters:
...
+## Deploying using CloudFormation to Amazon Managed Service for Apache Flink
+This sample assumes that MSK Serverless cluster is created. The flink application routes data ingested in source topic to sink topic without any transformation. 
+
+![Amazon Managed Service for Apache Flink , MSK Serverless example](images/flink-msk-serverless-example.png),
+### Pre-requisite
+1. Create MSK serverless cluster while choosing 3 subnets. Refer https://docs.aws.amazon.com/msk/latest/developerguide/serverless-getting-started.html . 
+2. Once the cluster is created note down subnets ids of the cluster and security group.
+3. Ensure that security group has self referencing ingress rule that allows connection on port 9098. 
+4. You have a user credential using which you can create CloudFormation stack from console or CLI. 
+
```
Comment (nicusX) [inline java/KafkaConnectors/README.md:87]: We should explain that, in order to run the application on the development machine, you need to access Kafka from your local machine, either

1. enable public access on MSK (but this is not supported in MSK Serverless)
2. run kafka locally, e.g. in containers

Explaining how to run Kafka in containers is out of scope. But if we want to make this an e2e example it should be e2e also when developing locally
We need at least to mention what is required

File: java/KinesisConnectors/src/main/resources/log4j2.properties [hunk 3]
```diff
@@ -1,4 +1,4 @@
-rootLogger.level = INFO
+rootLogger.level = WARN
```
Comment (nicusX) [inline java/KinesisConnectors/src/main/resources/log4j2.properties:1]: Let's keep INFO

File: java/WindowingTumbling/src/main/resources/log4j2.properties [hunk 4]
```diff
@@ -0,0 +1,7 @@
+rootLogger.level = WARN
```
Comment (nicusX) [inline java/WindowingTumbling/src/main/resources/log4j2.properties:1]: INFO

**Review by nicusX — CHANGES_REQUESTED** [review]
As we are making these examples e2e, they should be really e2e also when running locally.
We need at least to mention what you need to do with Kafka, i.e. enabling MSK public access (not available in MSK Serverless) or run Kafka locally.
No need to explain how to run Kafka locally (that will be too much) but at least mention

**Pull request merged**